sources/ConflictWindow.cpp

```diff
@@ -19,7 +19,7 @@ static const unsigned kReplace = 'RPLC';
 static const unsigned kSkip = 'SKIP';
 
 ConflictWindow::ConflictWindow(const char* file)
-	: BWindow(BRect(0, 0, 0, 0), "", B_BORDERED_WINDOW, B_WILL_ACCEPT_FIRST_CLICK | B_AUTO_UPDATE_SIZE_LIMITS)
+	: BWindow(BRect(0, 0, 0, 0), "", B_MODAL_WINDOW, B_WILL_ACCEPT_FIRST_CLICK | B_AUTO_UPDATE_SIZE_LIMITS)
```

Comment:
This is much better than B_BORDERED_WINDOW and B_MODAL_WINDOW. Thanks!
It seems that the floating window is not resizable even without B_NOT_RESIZABLE, so I have left it out. I have also removed the code that sets the Skip button as the default button because the new window can never receive keyboard focus.